packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -983,10 +982,10 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
         )
       case SBPaymentTypes.LINK_BANK:
         yield put(
-          actions.components.brokerage.showModal(
-            BrokerageModalOriginType.ADD_BANK,
-            fiatCurrency === 'USD' ? 'ADD_BANK_YODLEE_MODAL' : 'ADD_BANK_YAPILY_MODAL'
-          )
+          actions.components.brokerage.showModal({
```

Comment:
One snag I ran into is that RTK reducers enforce a `(state, payload) => {}` style for all reducers/actions so we can't pass in multiple arguments like we have done for the `showModal()` action in the past. Now the arguments must be passed as a single object (the payload), but this is was really easy to change throughout the app.